client/src/global/containers/Manifold/index.js

```diff
@@ -197,11 +214,13 @@ class ManifoldContainer extends PureComponent {
             active={this.props.visibility.signInUpOverlay}
             hideOverlay={hideOverlay}
           />
-          {fatalError.error ? (
+          {/* Add isRedirecting check here to ensure that FatalError doesn't trigger another redirect before we've finished navigating home on logout */}
+          {fatalError.error && !this.state.isRedirecting ? (
             <div className="global-container">
               <FatalError
                 dispatch={this.props.dispatch}
                 fatalError={fatalError}
+                redirectPath={this.props.location.pathname}
               />
```

Comment:
This is a weird one, but if you try on Edge to logout while viewing a project detail or one of its subpages, you don't get redirected home on logout but rather back to the project detail. I don't think this is intentional, and it's odd to only have the behavior on project pages. As far as I can tell, it's a side-effect of this component handling both the redirect and displaying errors. It seems like there's a render where the error still exists in the redux store after redirect has been called. That triggers this component to trigger a duplicate redirect.